client/app/components/FavoritesControl.jsx

```diff
@@ -31,6 +31,7 @@ export default class FavoritesControl extends React.Component {
     return (
       <a
         title={title}
+        aria-label={title}
```

Comment:
It seems in this case since the anchor already has a title it would say it twice? 🤔

Reply:
Unfortunately `title` is not widely accessible, so you will find some redundancies involving it in some bit of code. A more interesting approach would be to try to abstract the ambiguity of this pattern into `<Link>`, WDYT?

Reply:
I see, based on [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title#accessibility_concerns) it seems to me that we should avoid/drop the `title` use cases actually